README.md

```diff
@@ -74,6 +74,16 @@ DateTime strings in Azure Cosmos DB is `YYYY-MM-DDThh:mm:ss.fffffffZ` which foll
 
 ## Development
 
+### Git hooks
```

Comment:
Let's reference this section as a step in the **Setup**.
* Run `pre-commit install`. For more details, check out _Development > Git hooks_.